packages/components/styles/_forms.scss

```diff
@@ -1,9 +1,9 @@
+@import "~@kaizen/design-tokens/sass/color";
 @import "~@kaizen/design-tokens/sass/border";
 @import "~@kaizen/design-tokens/sass/typography";
 @import "./legacy/type";
```

Comment:
Is there anything that is used from `legacy/type` in this stylesheet? Most of those are prefixed by `ca-` 🤔 would be awesome if we didn't need it

Reply:
Nice catch! Nothing. Removed.